pyproject.toml

```
@@ -30,10 +30,11 @@ Jinja2 = "^3.0.3"
 sqlparse = ">=0.3.1,<0.5.0"
 SQLAlchemy = { version = "1.4.35", extras = ["mypy"] }
 feast = { version = "0.18.0", extras = ["redis", "gcp"], optional = true }
-Werkzeug = { version = "2.0.0", optional = true }                   # Bug no dash, com issue conhecida
+Werkzeug = { version = "2.0.0", optional = true }                          # Bug no dash, com issue conhecida
 dash = { version = "2.1.0", optional = true }
 dash-cytoscape = { version = "0.2.0", optional = true }
 protobuf = "<3.20.0"
+prometheus-fastapi-instrumentator = "^5.8.1"
```

```suggestion
prometheus-fastapi-instrumentator = { version = "^5.8.1", optional = true }
```

também precisa adicionar no `[tool.poetry.extras]`, ao lado do `feast`